includes/Frontend/Payment.php

```
@@ -161,208 +161,173 @@ public function payment_page( $content ) {
                 ?>
                 <?php
                 if ( count( $gateways ) ) {
+                    // Prepare order summary data before rendering
+                    $order_title = '';
+                    if ( $pack_id ) {
+                        $pack         = wpuf()->subscription->get_subscription( $pack_id );
+                        $details_meta = wpuf()->subscription->get_details_meta_value();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$details_meta'. (undefined)

(UnusedLocalVariable)